.github/workflows/actions.yml

```diff
@@ -18,7 +18,8 @@ jobs:
       matrix:
       # windows-latest is disabled due to cmake failure:
       # https://github.com/awvwgk/setup-fortran/issues/6
-        os: [ubuntu-latest, macos-latest]
+        os: [ubuntu-22.04, macos-12]
```

Comment:
being explicit about the runners, while ubuntu-22.04 is the latest, macos-12 is not the latest runner yet.
https://github.com/actions/runner-images

Reply:
released in 20.12.0.   

I'm not sure if this will require manual updating every time?